src/newTabs/ClassNames.tsx

```diff
@@ -0,0 +1,8 @@
+export class ClassNames {
```

Comment:
can't we update existing tab implementation ? or move newTab file in Tabs folder only so that we can reuse these classnames

Reply:
Done.